src/index/api.rs

```diff
@@ -70,10 +68,11 @@ fn get_release_time_yanked_downloads(pkg: &MetadataPackage) -> Result<(time::Tim
                 .get("created_at")
                 .and_then(|c| c.as_str())
                 .ok_or_else(|| err_msg("Not a JSON object"))?;
+
             release_time = Some(
-                time::strptime(release_time_raw, "%Y-%m-%dT%H:%M:%S")
+                DateTime::parse_from_str(release_time_raw, "%Y-%m-%dT%H:%M:%S%.f%:z")
                     .unwrap()
```

Comment:
It'd be nice to remove this unwrap so that we don't crash if the crates.io API returns something bad.